crs-setup.conf.example

```diff
@@ -654,19 +654,20 @@ SecDefaultAction "phase:2,log,auditlog,pass"
 # following directive somewhere after the inclusion of the CRS
 # (E.g., RESPONSE-999-EXCLUSION-RULES-AFTER-CRS.conf).
 #
-# SecRuleUpdateActionById 901450 "nolog"
+#SecRuleUpdateActionById 901450 "nolog"
```

Comment:
This line isn't explained in this file. The comment is in `REQUEST-901-INITIALIZATION.conf`:
```
# If a request is allowed to pass without being checked by the CRS, there is no
# entry in the audit log (for performance reasons), but an error log entry is
# being written.  If you want to disable the error log entry, then issue the
# following directive somewhere after the inclusion of the CRS
# (E.g., RESPONSE-999-EXCEPTIONS.conf).
```

That comment should be moved here, IMO. @dune73?